diff --git a/Qt7z/Qt7z/7zip/CPP/Common/MyBuffer.h b/Qt7z/Qt7z/7zip/CPP/Common/MyBuffer.h
index 818cdf4..d18afdb 100644
--- a/Qt7z/Qt7z/7zip/CPP/Common/MyBuffer.h
+++ b/Qt7z/Qt7z/7zip/CPP/Common/MyBuffer.h
@@ -5,6 +5,7 @@
 
 #include "Defs.h"
 #include "MyTypes.h"
+#include "common.h"
 
 /* 7-Zip now uses CBuffer only as CByteBuffer.
    So there is no need to use MY_ARRAY_NEW macro in CBuffer code. */
diff --git a/Qt7z/Qt7z/7zip/CPP/Common/MyVector.h b/Qt7z/Qt7z/7zip/CPP/Common/MyVector.h
index 0789d63..8714676 100644
--- a/Qt7z/Qt7z/7zip/CPP/Common/MyVector.h
+++ b/Qt7z/Qt7z/7zip/CPP/Common/MyVector.h
@@ -4,6 +4,7 @@
 #define __COMMON_MY_VECTOR_H
 
 #include <string.h>
+#include "common.h"
 
 template <class T>
 class CRecordVector
